pkg/schemes/scheme.go

```diff
@@ -64,11 +64,14 @@ func DefaultScheme() *runtime.Scheme {
 	return s
 }
 
-// TestingScheme unconditionally includes the default scheme and the gwv1a2 scheme (which includes TCPRoute).
+// TestingScheme unconditionally includes the default and required Gateway API schemes.
 func TestingScheme() *runtime.Scheme {
 	s := DefaultScheme()
 	if err := gwv1a2.Install(s); err != nil {
-		panic(fmt.Sprintf("Failed to install gwv1a2 scheme: %v", err))
+		panic(fmt.Sprintf("Failed to install gateway v1alpha2 scheme: %v", err))
```

Comment:
What are the circumstances where developers should follow the pattern of applying a scheme to the DefaultScheme, and when should they add it just to the TestingScheme? I think a comment outlining the standard could be helpful for the next developer, wdyt?

Reply:
I updated the `TestingScheme` name and included add'l godocs to provide better context in commit https://github.com/solo-io/gloo/commit/281ace000966aa7f15d960ab73d60efc4fa1c189.